.ci/Jenkinsfile

```diff
@@ -10,6 +10,8 @@ pipeline {
     DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
     PIPELINE_LOG_LEVEL='INFO'
     GO_VERSION = '1.13.12'
+    DOCKER_IMG = "${env.DOCKER_REGISTRY}/package-registry/distribution"
```

Comment:
@kuisathaverat for now, I just made this up. Not sure if this is going to work. 

The goal is that as soon as this PR is merged, it pushes to `docker.elastic.co/package-registry/distribution:snapshot`.

Reply:
currently, the pipeline uses the BRANCH_NAME as tag so it will push to `docker.elastic.co/package-registry/distribution:master`, we push for every branch or tag.

if we want to change it to `docker.elastic.co/package-registry/distribution:snapshot` for the master branch it is easy to make, Do we want to publish other branches to that namespace (`docker.elastic.co/package-registry`)? 

I see that there is a branch name snapshot, it push to `docker.elastic.co/package-registry/distribution:snapshot` but is not the master branch

Reply:
This PR is against the `snapshot` branch. Does this mean it is just going to work?

Reply:
ouch!!, I didn't realize that the PR is against the snapshot branch, yep it makes the thing